rna-germline-variant-calling.wdl

```diff
@@ -0,0 +1,777 @@
+  			known_indels_sites_indices = knownVcfsIndices,
+  			ref_dict = refDict,
+  			ref_fasta = refFasta,
+  			ref_fasta_index = refFastaIndex,
+  			preemptible_count = preemptible_count
+	}
+
+	call ApplyBQSR {
+		input:
+			input_bam = SplitNCigarReads.output_bam,
+			input_bam_index = SplitNCigarReads.output_bam_index,
+			base_name = sampleName + ".aligned.duplicates_marked.recalibrated",
+			ref_fasta = refFasta,
+			ref_fasta_index = refFastaIndex,
+			ref_dict = refDict,
+			recalibration_report = BaseRecalibrator.recalibration_report,
+			compression_level = compression_level,
+			preemptible_count = preemptible_count
+	}
+
+	call ScatterIntervalList {
+		input:
+			interval_list = wgsCallingIntervalList,
+      		scatter_count = haplotypeScatterCount,
+      		break_bands_at_multiples_of = breakBandsAtMultiplesOf,
+      		preemptible_count = preemptible_count
+	}
+
+    scatter (i in range(ScatterIntervalList.interval_count)) {
+		call HaplotypeCaller {
+			input:
+				input_bam = ApplyBQSR.output_bam,
+				input_bam_index = ApplyBQSR.output_bam_index,
+				base_name = sampleName + ".hc",
+				interval_list = ScatterIntervalList.out[i],
+				ref_fasta = refFasta,
+				ref_fasta_index = refFastaIndex,
+				ref_dict = refDict,
+				preemptible_count = preemptible_count
+		}
+	}
+
+	call MergeVCFs {
+    	input:
+      		input_vcfs = HaplotypeCaller.output_gvcf,
+      		input_vcfs_indexes = HaplotypeCaller.output_gvcf_index,
+      		output_vcf_name = sampleName + ".g.vcf.gz",
+      		preemptible_count = preemptible_count
+  	}
+
+  	call GenotypeGVCFs {
+  		input:
+  			input_gvcf = MergeVCFs.output_vcf,
+  			input_gvcf_index = MergeVCFs.output_vcf_index,
+  			output_vcf_filename = sampleName + ".vcf.gz",
+  			dbSNP_vcf = dbSnpVcf,
+  			dbSNP_vcf_index = dbSnpVcfIndex,
+  			ref_fasta = refFasta,
+  			ref_fasta_index = refFastaIndex,
+  			ref_dict = refDict,
+  			interval_list = wgsCallingIntervalList,
+  			preemptible_count = preemptible_count
+  	}
+
+	call VariantFiltration {
+		input:
+			input_vcf = GenotypeGVCFs.output_vcf,
+			input_vcf_index = GenotypeGVCFs.output_vcf_index,
+			base_name = sampleName + ".variant_filtered.vcf.gz",
+			ref_fasta = refFasta,
+			ref_fasta_index = refFastaIndex,
+			ref_dict = refDict,
+			preemptible_count = preemptible_count
+	}
+
+	output {
+		File recalibrated_bam = ApplyBQSR.output_bam
+		File recalibrated_bam_index = ApplyBQSR.output_bam_index
+		File merged_gvcf = MergeVCFs.output_vcf
+		File merged_gvcf_index = MergeVCFs.output_vcf_index
+		File variant_filtered_gvcf = VariantFiltration.output_vcf
+		File variant_filtered_gvcf_index = VariantFiltration.output_vcf_index
+	}
+}
+
+task SortSam {
+	File unsorted_bam
+	String base_name
+	String sort_order
+
+	Int preemptible_count
+
+	command <<<
+	 	java -jar /usr/gitc/picard.jar \
+	 	    SortSam \
+	 	    INPUT=${unsorted_bam} \
+	 	    OUTPUT=${base_name}.bam \
+	 	    SORT_ORDER=${sort_order} \
+	 	    COMPRESSION_LEVEL=0 \
```

Comment:
(Do keep this line -- it's the other "compressions" that should be removed and we should use default)
However, you should change this to level=1.